portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path

```diff
@@ -117,7 +117,7 @@
 </tr>
 <tr>
 	<td>ANY_WITH_ARIA_LABEL_INDEX</td>
-	<td>xpath=(//button[@aria-label='${ariaLabel}'])[${key_index}]</td>
+	<td>xpath=(//button[contains(@aria-label, 'Remove Duplicate Field Fields Group')])[${key_groupIndex}]</td>
```

Comment:
Since this Path is "ANY_....", it should not be hard coded to a single aria-label value. You can pass this value by the macro, as it's being done already and not being used.